src/main/java/mat/server/CQLServiceImpl.java

```diff
@@ -414,8 +414,7 @@ public SaveUpdateCQLResult saveAndModifyFunctions(String xml, CQLFunctions funct
             result.getCqlModel().setCqlFunctions(sortFunctionList(result.getCqlModel().getCqlFunctions()));
             result.setSuccess(true);
         } catch (Exception e) {
-            e.printStackTrace();
-            result.setSuccess(false);
+            logger.error(e.getMessage(), e);

```

Comment:
Should the `result.setSuccess(false);` be left in? I saw it was left in other methods.

Reply:
Well, it's false by default. But, maybe I will leave it for clarify

Reply:
good catch